security-analysis/security-analysis-api/src/main/java/com/powsybl/security/action/ActionList.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.action;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.powsybl.commons.json.JsonUtil;
+import com.powsybl.security.json.SecurityAnalysisJsonModule;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author Etienne Lesot <etienne.lesot@rte-france.com>
+ */
+public class ActionList {
+    private final List<Action> actions;
+
+    public ActionList(List<Action> actions) {
+        this.actions = Objects.requireNonNull(actions);
+    }
+
+    public List<Action> getActions() {
+        return actions;
+    }
+
+    public static ActionList readFile(Path jsonFile) {
+        try (InputStream is = Files.newInputStream(jsonFile)) {
+            return readInputStream(is);
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+    }
+
+    public static ActionList readInputStream(InputStream is) {
+        Objects.requireNonNull(is);
+        try {
+            return createObjectMapper().readValue(is, ActionList.class);
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+    }
+
+    /**
+     * Writes action list as JSON to a file.
+     */
+    public static void writeFile(ActionList actionList, Path jsonFile) {
```

Comment:
We can now make it non-static and remove the first argument (same for other write methods)